tests/gnrc_dhcpv6_client/Makefile

```diff
@@ -41,6 +41,8 @@ include $(RIOTBASE)/Makefile.include
 
 .PHONY: dhcpv6_server
 
+test-with-config/check-config: IFACE := ${IFACE}
```

Comment:
> somehow IFACE sems to not be set
> 
> ```
> tests/gnrc_dhcpv6_client$ make test-with-config
> Device "" does not exist.
> You may be able to create "" by using e.g. `dist/tools/tapsetup/tapsetup`.
> System configuration for tests is not available, check tests/gnrc_dhcpv6_client/tests-with-config/check-config.sh failed.
> ```
> 
> ```
> tests/gnrc_dhcpv6_client$ IFACE=tapbr0 make test-with-config
> ```
> 
> finishes successfully

Seems like this does not work. Any idea, what is happening @fjmolinas?

Reply:
you may need export

Reply:
Maybe `$(call target-export-variables,test-with-config/check-config,IFACE)`

Reply:
You can remove the export for the rule, using this target is actually what I meant (but was absolutely not clear about) in previous PR

Reply:
```
make -C tests/gnrc_dhcpv6_client test-with-config
/home/francisco/workspace/RIOT2/tests/gnrc_dhcpv6_client/dhcpv6_server.sh 61342 /home/francisco/workspace/RIOT2/tests/gnrc_dhcpv6_client/kea-dhcp6.conf
/usr/sbin/kea-dhcp6
2021-09-07 10:52:47.025 INFO  [kea-dhcp6.dhcp6/569391] DHCP6_STARTING Kea DHCPv6 server version 1.6.2 starting
2021-09-07 10:52:47.036 INFO  [kea-dhcp6.hosts/569391] HOSTS_BACKENDS_REGISTERED the following host backend types are available: mysql postgresql 
2021-09-07 10:52:47.037 INFO  [kea-dhcp6.dhcpsrv/569391] DHCPSRV_CFGMGR_ADD_IFACE listening on interface tapbr0
2021-09-07 10:52:47.037 INFO  [kea-dhcp6.dhcpsrv/569391] DHCPSRV_CFGMGR_NEW_SUBNET6 a new subnet has been added to configuration: 2001:db8::/32 with params: t1=1000, t2=2000, preferred-lifetime=3000, valid-lifetime=4000, rapid-commit is false
2021-09-07 10:52:47.037 INFO  [kea-dhcp6.dhcp6/569391] DHCP6_CONFIG_COMPLETE DHCPv6 server has completed configuration: added IPv6 subnets: 1; DDNS: disabled
2021-09-07 10:52:47.037 INFO  [kea-dhcp6.dhcpsrv/569391] DHCPSRV_MEMFILE_DB opening memory file lease database: name=kea-dhcp6.leases.csv type=memfile universe=6
2021-09-07 10:52:47.038 INFO  [kea-dhcp6.dhcpsrv/569391] DHCPSRV_MEMFILE_LEASE_FILE_LOAD loading leases from file kea-dhcp6.leases.csv
2021-09-07 10:52:47.038 INFO  [kea-dhcp6.dhcpsrv/569391] DHCPSRV_MEMFILE_LFC_SETUP setting up the Lease File Cleanup interval to 3600 sec
2021-09-07 10:52:47.038 INFO  [kea-dhcp6.dhcp6/569391] DHCP6_USING_SERVERID server is using server-id 00:01:00:01:28:c9:e4:df:7c:50:79:8b:6f:78 and stores in the file ./kea-dhcp6-serverid
2021-09-07 10:52:47.038 INFO  [kea-dhcp6.dhcp6/569391] DHCP6_STARTED Kea DHCPv6 server version 1.6.2 started
r
/home/francisco/workspace/RIOT2/tests/gnrc_dhcpv6_client/bin/native/tests_gnrc_dhcpv6_client.elf tap0 
RIOT native interrupts/signals initialized.
LED_RED_OFF
LED_GREEN_ON
RIOT native board initialized.
RIOT native hardware initialization complete.

NETOPT_RX_END_IRQ not implemented by driver
Help: Press s to start test, r to print it is ready
READY
s
START
main(): This is RIOT! (Version: 2021.10-devel-584-g845d58-pr-16797)
Iface  5  HWaddr: 5A:79:C4:9E:A0:74 
          L2-PDU:1500  MTU:1500  HL:64  Source address length: 6
          Link type: wired
          inet6 addr: fe80::5879:c4ff:fe9e:a074  scope: link  VAL
          inet6 group: ff02::1
          inet6 group: ff02::1:ff9e:a074
          
2021-09-07 10:52:49.286 INFO  [kea-dhcp6.leases/569391] DHCP6_LEASE_ADVERT duid=[00:03:00:01:5a:79:c4:9e:a0:74], tid=0xc30a4: lease for address 2001:db8:1:: and iaid=196613 will be advertised
2021-09-07 10:52:49.286 INFO  [kea-dhcp6.leases/569391] DHCP6_PD_LEASE_ADVERT duid=[00:03:00:01:5a:79:c4:9e:a0:74], tid=0xc30a4: lease for prefix 2001:db8:8000::/64 and iaid=1638405 will be advertised
2021-09-07 10:52:50.353 INFO  [kea-dhcp6.leases/569391] DHCP6_LEASE_ALLOC duid=[00:03:00:01:5a:79:c4:9e:a0:74], tid=0x14e367: lease for address 2001:db8:1::1 and iaid=196613 has been allocated for 4000 seconds
2021-09-07 10:52:50.353 INFO  [kea-dhcp6.leases/569391] DHCP6_PD_LEASE_ALLOC duid=[00:03:00:01:5a:79:c4:9e:a0:74], tid=0x14e367: lease for prefix 2001:db8:8000:1::/64 and iaid=1638405 has been allocated for 4000 seconds
Iface  5  HWaddr: 5A:79:C4:9E:A0:74 
          L2-PDU:1500  MTU:1500  HL:64  Source address length: 6
          Link type: wired
          inet6 addr: fe80::5879:c4ff:fe9e:a074  scope: link  VAL
          inet6 addr: 2001:db8:1::1  scope: global  VAL
          inet6 addr: 2001:db8:8000:1:5879:c4ff:fe9e:a074  scope: global  VAL
          inet6 group: ff02::1
          inet6 group: ff02::1:ff9e:a074
          inet6 group: ff02::1:ff00:1
          
2001:db8:1::1/128 dev #5  expires 3996 sec deprecates 2996 sec
2001:db8:8000:1::/64 dev #5  expires 3996 sec deprecates 2996 sec
```

Reply:
Done as proposed.